index.html

```diff
@@ -6,6 +6,51 @@
 
   <title>My Resume</title>
 
+  <div class='navigation'>
+    <a class='items' href="#">Intro</a>
+    <a class='items' href="#">Skills</nav>
+    <a class='items' href="#">Work History</a>
+    <a class='items' href="#">Contact</a>
+  </div>  
+<hr>  
+
+  <div class='intro'>
+    <p>Hello my name is</p>
+  </div>
+
+  <div class='Name'>
+      <h1>Michael Baynon</h1>
+  </div>
+
+  <div class='desc'>
+    <p>and I am a Full Stack Web Developer!<p>
+  </div>
+<hr>
+  <div class='introStatement'>
+    <h3>Why hire me?</h3>
+    <p class='introStatementText'>I would be a good hire because I am knowledgable, proffessional, and extremely hard working. In addition I am creative and willing to learn new things for the job. </p>
+  </div>
+<hr>
+  <div class='skills'>
+    <h3>My Skills</h3>
+    <p class='skillsText'>My skills include HTML, CSS, Javascript, and much more!</p> 
+  </div>
+<hr>
+  <div class='workHistory'>
+    <h3>Work History</h3>
+    <p class='workHistoryText'>Apple, Google, Amazon, and many more!</p>
+  </div>
+<hr>
+  <div class='contact'>
+    <h3>Contact me</h3>
+    
+    <div class='contactText'>
+      <p>firstNameLastName@gmail.com</p>
+    <br>  
```

Comment:
Same comment as the `hr` above. These are quick and dirty solutions when you have no choice, or other way of making it work. The `hr` should be a border bottom, the `br` here a margin between those `p` tags (or flex the parent container to space them.) Leave all the styling of the page to be in the CSS and all the Mark(ed)up content(the M in HTML) to be in the HTML files.